.github/workflows/linux-eic-shell.yml

```
@@ -266,7 +267,7 @@ jobs:
         llvm-profdata-15 merge -sparse src/tests/algorithms_test/algorithms_test.profraw \
           -o src/tests/algorithms_test/algorithms_test.profdata
         LIB_PATHS=()
-        for LIB_NAME in algorithms_calorimetry algorithms_digi algorithms_pid algorithms_reco algorithms_tracking; do
+        for LIB_NAME in algorithms_calorimetry algorithms_digi algorithms_pid algorithms_pid_lut algorithms_reco algorithms_tracking; do
```

Fragile and probably should be programmatically derived.

---

We have src/algorithms/interfaces and src/algorithms/meta that are not even libraries. If you know of a clean way, let me know.

---

Would something like this work, or is it too broad?
```suggestion
        for LIB_NAME in $(find src/ -type f -name "libalgorithms_*.so"); do
```
Also makes the next line obsolete.

---

I think this can work, as long as libraries don't disappear from the build. I've implemented.